src/cmd/linuxkit/moby/config.go

```diff
@@ -110,12 +110,13 @@ type ImageConfig struct {
 
 // Runtime is the type of config processed at runtime, not used to build the OCI spec
 type Runtime struct {
-	Cgroups    *[]string      `yaml:"cgroups,omitempty" json:"cgroups,omitempty"`
-	Mounts     *[]specs.Mount `yaml:"mounts,omitempty" json:"mounts,omitempty"`
-	Mkdir      *[]string      `yaml:"mkdir,omitempty" json:"mkdir,omitempty"`
-	Interfaces *[]Interface   `yaml:"interfaces,omitempty,omitempty" json:"interfaces,omitempty"`
-	BindNS     Namespaces     `yaml:"bindNS,omitempty" json:"bindNS,omitempty"`
-	Namespace  *string        `yaml:"namespace,omitempty" json:"namespace,omitempty"`
+	Cgroups     *[]string          `yaml:"cgroups,omitempty" json:"cgroups,omitempty"`
+	Mounts      *[]specs.Mount     `yaml:"mounts,omitempty" json:"mounts,omitempty"`
+	Mkdir       *[]string          `yaml:"mkdir,omitempty" json:"mkdir,omitempty"`
+	Interfaces  *[]Interface       `yaml:"interfaces,omitempty,omitempty" json:"interfaces,omitempty"`
+	BindNS      Namespaces         `yaml:"bindNS,omitempty" json:"bindNS,omitempty"`
+	Namespace   *string            `yaml:"namespace,omitempty" json:"namespace,omitempty"`
+	EnvFromFile *map[string]string `yaml:"envFromFile,omitempty" json:"envFromFile,omitempty"`
```

Comment:
Also, I'm not sure if this has to be a map - `env` is a list, but this has different meaning really. I'm not feeling too strongly about it and happy to change.

Reply:
Perhaps we should change it ot `envFromFiles` actually? ...unless someone has ideas for even better name?

Reply:
yeah `envFromFiles` probably better. I think a map is best, we try to avoid microformats...

Reply:
Sure, happy to make it plural and go with a map.